pkg/integrations/v4/supervisor_fb.go

```diff
@@ -19,7 +24,11 @@ import (
 	"github.com/newrelic/infrastructure-agent/pkg/sysinfo/hostname"
 )
 
-var sFBLogger = log.WithComponent("integrations.Supervisor").WithField("process", "log-forwarder")
+var (
+	//nolint:gochecknoglobals
+	sFBLogger              = log.WithComponent("integrations.Supervisor").WithField("process", "log-forwarder")
+	luaFilterTempFileRegex = regexp.MustCompile(`nr_fb_lua_filter\d+`)
```

Comment:
In the original code the regular expression was set as `regexp.MustCompile("nr_fb_lua_filter\\d+")`,. with double backslash. Can you double-check to ensure your pattern is correct, please?

Reply:
I changed it to backquotes as suggested by the linter :) You don't need to escape it twice with backquotes  https://go.dev/play/p/JXh0gGAAaf5